### DISCUSSION TITLE
Temps archives on HDD with plots

### DISCUSSION THREAD

**Discussion by Zillos** [Chia-Network/chia-blockchain#11399] [Support]
![WhatsApp Image 2022-05-02 at 5 45 15 PM](https://user-images.githubusercontent.com/101582623/166334045-01f52bf0-bc51-402c-82c6-a54933d6611e.jpeg)

Is this normal? to have temps archives on HDD dedicated to store plots?... is safe to delete them? it affect the plotting and farming system?

Thank you!

**Comment by loppefaaret** [comment]
since it is not a `.plot` file, i assume it is a possible incomplete plot file, that failed during processing it... if so, it is safe to remove, and replace
